doc/examples/histology_resdnn.py

```diff
@@ -59,8 +59,7 @@
 val = unique[np.argmax(count[1:])+1]
 mask[mask_labeled != val] = 0
 
-nib.save(nib.Nifti1Image(mask.astype(np.uint8), dwi_img.affine),
-         'mask.nii.gz')
+save_nifti('mask.nii.gz', mask.astype(np.uint8), affine)
```

Comment:
I really don't like that function, I would rather avoid it. I prefer using Nibabel directly.